scripts/sct_register_multimodal.py

```diff
@@ -764,6 +792,17 @@ def register(src, dest, paramreg, param, i_step_str):
                            ants_registration_params=ants_registration_params,
                            remove_temp_files=param.remove_temp_files,
                            verbose=param.verbose)
+        else:  # im_seg case
```

Comment:
as we agreed in a previous comment, please explicit the use of HOG to prevent confusion in the future.